CMakeLists.txt

```diff
@@ -11,11 +11,19 @@ set(CUKE_ENABLE_EXAMPLES OFF CACHE BOOL "Enable the examples")
 set(GMOCK_DIR "" CACHE STRING "Google Mock framework sources path (otherwise downloaded)")
 set(GMOCK_VER "1.7.0" CACHE STRING "Google Mock framework version to be used")
 set(VERBOSE OFF CACHE BOOL "Verbose output")
+set(APPVEYOR OFF CACHE BOOL "Is this appveyor CI build")
```

Comment:
Ok, I'll remove it. I suppose /MD->MT flag change is needed with MSVC regardles of appveyor or not